gallery/src/components/NxDrawer/NxDrawerPage.tsx

```diff
@@ -0,0 +1,215 @@
+            <NxH3>NxDrawer.Header (Required)</NxH3>
+          </NxTile.SubsectionHeader>
+          <NxP>
+            The header is placed at the top of the drawer component. It consists of four parts:
+            the close button, the title, the subtitle (optional), and the description (optional).
+            The title, subtitle, and description can be specified via the convenience components:{' '}
+            the <NxCode>NxDrawer.Header.Title</NxCode>, <NxCode>NxDrawer.Header.Subtitle</NxCode>,{' '}
+            and <NxCode>NxDrawer.Header.Description</NxCode>, respectively.
+          </NxP>
+          <NxP>
+            If the <NxCode>NxDrawer.Header.Title</NxCode> is too long to be fully seen, it will be truncated.
+            A tooltip will be visible when hovering over the title to read the full title.
+          </NxP>
+        </NxTile.Subsection>
+        <NxTile.Subsection>
+          <NxTile.SubsectionHeader>
+            <NxH3>NxDrawer.Content (Required)</NxH3>
+          </NxTile.SubsectionHeader>
+          <NxP>
+            This component specifies the drawer's body content. It is placed
+            under the header and extends the full height of the drawer if the footer is not specified.
+            If the footer is specified, it occupies the height between the header and the footer.
+            The content area will be scrollable if the content overflows.
+          </NxP>
+        </NxTile.Subsection>
+        <NxTile.Subsection>
+          <NxTile.SubsectionHeader>
+            <NxH3>NxFooter (Optional)</NxH3>
+          </NxTile.SubsectionHeader>
+          <NxP>
+            This is an optional component that sets the drawer's footer content. It is placed
+            at the bottom and it takes the height of its content.
+          </NxP>
+          <NxP>
+            Alternatively, the &lt;footer&gt; element can be used with the class <NxCode>nx-footer</NxCode>.
+          </NxP>
+        </NxTile.Subsection>
+        <NxTile.Subsection>
+          <NxTile.SubsectionHeader>
+            <NxH3>Accessibility</NxH3>
+          </NxTile.SubsectionHeader>
+          <NxP>
+            <NxCode>NxDrawer</NxCode> implements the dialog role, similar to <NxCode>NxModal</NxCode>.
+            It is recommended that <NxCode>aria-labeledby</NxCode> or <NxCode>aria-label</NxCode> should be specified.
+            Please refer to <NxTextLink href="#/pages/Modal">the Accessibility Section</NxTextLink> of the{' '}
+            <NxCode>NxModal</NxCode> documentation.
+          </NxP>
+        </NxTile.Subsection>
+      </GalleryDescriptionTile>
+
+      <GalleryExampleTile title="Simple NxDrawer Example"
+                          id="nx-drawer-simple-example"
+                          codeExamples={NxDrawerSourceCode}>
+        A basic example of an <NxCode>NxDrawer</NxCode> with and without a footer.
+        This example also shows a NxDrawer with overflowing content.
+        <br/>
+        <br/>
```

Comment:
Don't use multiple break tags for vertical spacing.  Observe how the Global Header page lays this out.

Reply:
> Don't use multiple break tags for vertical spacing. Observe how the Global Header page lays this out.

Fixed here: e79da15124b174eb9252c52da95f410ff3a5b8b1

Reply:
> > Don't use multiple break tags for vertical spacing. Observe how the Global Header page lays this out.
> 
> Fixed here: [e79da15](https://github.com/sonatype/sonatype-react-shared-components/commit/e79da15124b174eb9252c52da95f410ff3a5b8b1)

Updated formatting here: 1f0d94eda1bef809423f7b25a1184eb8bd717759